tests/components/http/scenarios/response_send_status.js

```diff
@@ -0,0 +1,49 @@
+const { assert_log } = require('../../../scripts/operators.js');
+const { HyperExpress, fetch, server } = require('../../../configuration.js');
+const router = new HyperExpress.Router();
```

Comment:
This won't be necessary I think if we use `TEST_SERVER` for creating the API handler